general/general-contributing.asciidoc

```diff
@@ -1,15 +1,11 @@
 = Contributing Guide
 
-include::Contributing-Code.asciidoc[leveloffset=1]
+include::../.github/CONTRIBUTING.asciidoc[leveloffset=1]
 
-include::Contributing-Code-of-Conduct.asciidoc[leveloffset=1]
-
-include::Contributing-Development-Guidelines.asciidoc[leveloffset=1]
+include::../.github/CODE_OF_CONDUCT.md[leveloffset=1]
```

Comment:
Usually we don't include files from a submodules directly. The convention is that every submodule must have its own master file (master-devon4ng for devon4ng, master-devon4j for devon4j, ...) so you have to create a "master-github" file and include CONTRIBUTING and CODE_OF_CONDUCT from them. Then you can include master-github from master.asciidoc (the one in devonfw-guide)

Reply:
Yes, that would be a cleaner solution, I agree. But I think all files in the `.github` repo will be included in all (new) repos of the org by default, so we might have to keep the contents of `.github` as minimal as possible ... I'll have to ask @hohwille what he thinks about this special case and how we should proceed.